packages/core/settings/_colours.scss

```diff
@@ -92,39 +90,34 @@ $color_shade_nhsuk-blue-50: shade($color_nhsuk-blue, 50%); // [4] //
 $color_shade_nhsuk-green-35: shade($color_nhsuk-green, 35%); // [5] //
 $color_shade_nhsuk-green-50: shade($color_nhsuk-green, 50%); // [6] //
 
-$color_tint_nhsuk-warm-yellow-30: tint($color_nhsuk-warm-yellow, 30%); // [7] //
-$color_tint_nhsuk-warm-yellow-10: tint($color_nhsuk-warm-yellow, 10%); // [8] //
```

Comment:
We need to document this change and update the service manual.

Also need document which colour variable people should use instead.